SR2020/2020-03-05-coronavirus.md

```diff
@@ -0,0 +1,22 @@
+---
+to: sr2020-teams
+subject: Update on Coronavirus and SR
```

Comment:
nit: I'm not sure how well known it is that "SR" == "Student Robotics" outside of volunteers. I suggest for clarity we expand the acronym, at least in the subject line.
```suggestion
subject: Update on Coronavirus and Student Robotics
```